src/index.js

```diff
@@ -233,7 +190,7 @@
 
       if (isBrowser()) {
         styles.push(match.rule.style)
-        styles.push('color:inherit;')
+        styles.push('') // Empty string resets style.
```

Comment:
But `styles.push('') `happens only once. The others are unrelated.
But if it's not clear, let me know.
If the code itself is not clear, we have to put comments.